medicines/web/README.md

```diff
@@ -8,23 +8,7 @@ A React web application.
 
 The site is configured to run locally at [`http://localhost:3000/`](http://localhost:3000/)
 
-You will need to set the local environment (variables provided below, see [the Azure pipeline definition](/azure-pipelines.yml) for good default values).
-
-```
-export AZURE_SEARCH_API_VERSION=
-export AZURE_SEARCH_EXACTNESS_BOOST=
-export AZURE_SEARCH_INDEX=
-export AZURE_SEARCH_KEY=
-export AZURE_SEARCH_SCORING_PROFILE=
-export AZURE_SEARCH_SERVICE=
-export AZURE_SEARCH_WORD_FUZZINESS=
-export GOOGLE_GTM_CONTAINER_ID=
-export GOOGLE_TRACKING_ID=
-export GOOGLE_USE_DEBUG=
-export ROOT_URL_DOMAIN=
-```
-
-We recommend that you set these in your `.profile` or `.bashrc` file so that it persists past the lifetime of your terminal session.
+You will need to get the local environment variables by running `make get-env`.
```

Comment:
We need to create a guide for setting up the keyvault on their Azure for developers without access to production values who want to run the code.